docs/getting-started/clean-up.md

```diff
@@ -0,0 +1,34 @@
+## Clean Up
+
+=== "kn"
+    Delete the Service
+    ```bash
+    kn service delete cloudevents-player
+    ```
+
+    Delete the Trigger
+    ```bash
+    kn trigger delete cloudevents-trigger
+    ```
+
+=== "kubectl"
+    Delete the Service
+    ```bash
+    kubectl delete -f cloudevents.yaml
+    ```
+
+    Delete the Trigger
+    ```bash
+    kubectl delete -f ce-trigger.yaml
+    ```
+
+Delete the Cluster
```

Comment:
1. Delete the cluster by running the command:

Reply:
I get that this is a Style Guide thing, but it looks weird as an admonition

Reply:
ah, saw your next comment